### DISCUSSION TITLE
扩展倍速失效

### DISCUSSION THREAD

**Discussion by zyj13547810610** [the1812/Bilibili-Evolved#3559] [提问 / Q&A]
扩展倍数和记忆倍数在登录账号后失效，不知道是不是只有我一个人出现这种问题

**Comment by zyj13547810610** [comment]
准确的说是出在cookie上，删除cookie后又可以了，但是一刷新又失效

**Reply by zyj13547810610** [reply]
我虚拟机重新开了个edge依旧失效，看样子edge的问题，chrome正常使用

**Reply by pkpkth** [reply]
我的chrome也失效了

**Reply by zyj13547810610** [reply]
那我就不太清楚了
😂

**Comment by the1812** [comment]
#3187

**Reply by Flying-Fish-18** [reply]
兄弟有没有找到解决方法。

**Reply by zyj13547810610** [reply]
> 兄弟有没有找到解决方法。

edge 是搜索栏旁边那个🔒的图标，点开有个cookie   里面把 www.bilibili.com 删除，不过每次进入网站都需要重复删除
